en_US.json

```diff
@@ -1844,6 +1844,16 @@
 	"report.general.button.download.raw.data": "Download Raw Data",
 	"report.general.label.cache.hit.ratio": "Cache Hit Ratio",
 	"report.general.label.subtitle.zoom.in": "Click and drag in the plot area to zoom in",
+	"report.range.error.msg.earlier.than": "Range End cannot be earlier than the Range Start time.",
+	"report.range.error.msg.later.than": "The Range Start cannot be later than the Range End time.",
```

Comment:
@chiraags Thanks.
Yes, you're right. I was trying to show each message when selected each range start and end. I thought it would be a little different, but if I think the meaning of the two is the same, I think it would be right to reduce it to one.

Reply:
I removed the same meaning.
Thanks.